components/atomic/index.ts

```diff
@@ -7,7 +7,7 @@ export { default as Dropdown } from "./dropdown/Dropdown";
 export { default as T } from "./Translate";
 export { default as DataViewToggle } from "./DataViewToggle";
 export { default as Image } from "./Image";
-export { default as ProgressBar } from "./ProgressBar";
+export { default as ProgressBar } from "./loading/ProgressBar";
```

Comment:
@1aurend Rebasing off main doesn't squash any commits, instead it tries to resolve conflits and put your commits on top of the latest changes from main. I rebased your branch and it looks good! I did notice that you're missing the LoadingCircle export here in atomic components.